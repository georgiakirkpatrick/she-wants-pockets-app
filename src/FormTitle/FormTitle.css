* {
    box-sizing: border-box;
    padding: 0;
    margin: 0;
}

.FormTitle {
    font-size: 30px;
    text-align: center;
}