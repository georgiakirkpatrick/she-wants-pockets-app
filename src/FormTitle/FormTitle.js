import React from 'react'
import './FormTitle.css'

const FormTitle = props => {
    return (
        <h2 className='FormTitle'>
            {props.titleText}
        </h2>
    )
}

export default FormTitle