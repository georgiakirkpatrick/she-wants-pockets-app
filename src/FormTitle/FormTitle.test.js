import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import FormTitle from './FormTitle'

describe('FormTitle', () => {
        it ('renders without crashing', () => {
            const div = document.createElement('div')
            ReactDOM.render(<FormTitle />, div)
            ReactDOM.unmountComponentAtNode(div)
        })
    
        it ('render the UI as expected', () => {
            const tree = renderer
                .create(<FormTitle />)
                .toJSON()
            expect(tree).toMatchSnapshot()
        })
    })