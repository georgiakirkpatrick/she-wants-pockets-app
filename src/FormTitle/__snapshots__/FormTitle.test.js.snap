// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormTitle render the UI as expected 1`] = `
<h2
  className="FormTitle"
/>
`;
